fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/item/InventoryStorage.java

```diff
@@ -69,11 +69,21 @@ static InventoryStorage of(Inventory inventory, @Nullable Direction direction) {
 	 * Retrieve an unmodifiable list of the wrappers for the slots in this inventory.
 	 * Each wrapper corresponds to a single slot in the inventory.
 	 */
+	@Override
 	List<SingleSlotStorage<ItemVariant>> getSlots();
 
 	/**
-	 * Retrieve a wrapper around a specific slot of the inventory.
+	 * {@inheritDoc}
 	 */
+	@Override
+	default int getSlotCount() {
+		return getSlots().size();
```

Comment:
Isn't that a problematic recursion since `getSlots` uses `getSlotCount` ?

Reply:
`InventoryStorage` reabstracts `getSlots`, and the two implementations override it anyway.